providers/caching/redis.js

```diff
@@ -125,12 +127,19 @@ class RedisCache {
    *
    * @param {RedisCacheOptions} options - Redis connection configuration
    * @returns {RedisClientType} A new Redis client instance
-   */
+  //  */
   static buildRedisClient({ apiKey, service, port = 6380, tls = true }) {
+    const socketOptions = { host: service, port }
+
+    if (tls) {
+      // @ts-ignore
```

Comment:
Can you make tls an optional property in the type you define above? Or just make `socketOptions` of type `RedisSocketOptions`?